src/main/java/io/synadia/flink/source/NATSConsumerConfig.java

```diff
@@ -0,0 +1,48 @@
+package io.synadia.flink.source;
```

Comment:
Okay, refactored to `Nats`

Reply:
I totally agree with the idea of repackaging.
For an easy review of this PR, I will make those changes in the next PR as it would involve reviewing a lot of file changes.

Reply:
Alternatively we could put it in a different commit if @scottf is fine to review by picking out commits individually.